packages/spark-core/components/_pagination.scss

```diff
@@ -8,33 +8,71 @@
   justify-content: center;
   margin-right: auto;
   margin-left: auto;
-  max-width: 26rem;
+  max-width: $pag-max-width;
   width: 100%;
 }
 
-.sprk-c-Pagination__item {
-  margin-right: $space-m;
-  padding: $space-inset-short-s;
+// Pag links have a border when hovered
+.sprk-c-Pagination__link {
+  color: $black;
+  border: 2px solid transparent;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: $pag-circle-size;
+  height: $pag-circle-size;
+  text-align: center;
+  text-decoration: none;
 
-  &:last-child {
-    margin-right: 0;
+  &:hover,
+  &:focus,
+  &:active,
+  &:visited {
+    color: $black;
+  }
+
+  &:hover,
+  &:focus,
+  &:active {
+    border-color: $pag-selection-color;
   }
 }
 
-// For items in the long pagination version
-.sprk-c-Pagination__item--long {
-  margin-right: $space-xs;
+.sprk-c-Pagination__link--current {
+  background-color: $pag-selection-color;
+  color: $white;
```

Comment:
this should be a var too probably, cause if they can set $pag-selection-color to something else, they might need to change the color too.

Reply:
ah yeh! Good thinking.